app/test/main_test.dart

```diff
@@ -16,6 +17,7 @@ void main() {
     testWidgets('first route is welcome screen', (WidgetTester tester) async {
       addLazySingletonForTest(testLocator, () => WelcomeViewModel());
       addLazySingletonForTest(testLocator, () => NavigationService());
+      addLazySingletonForTest(testLocator, () => DialogService());
```

Comment:
Do we need this? I think the current tests do not use `Sign up` code.

Reply:
We need this as I used it in signInViewModel. That is just to show how to use the dialog service. I thought it could be helpful to show at least 1 example showing how to use the service